framework/platform/bpl/uci/db/bpl_db_uci.cpp

```diff
@@ -389,6 +389,34 @@ bool uci_get_all_sections(const std::string &package_name, const std::string &se
     //package_name.(section_type)
     LOG(TRACE) << "uci_get_all_sections() " << package_name << ".(" << section_type << ")";
 
+    auto ctx = alloc_context();
+    if (!ctx) {
+        return false;
+    }
+
+    char path[MAX_UCI_BUF_LEN] = {0};
+    if (snprintf(path, MAX_UCI_BUF_LEN, package_path, package_name.c_str()) <= 0) {
+        LOG(ERROR) << "Failed to compose path";
+        return false;
+    }
+
+    struct uci_ptr ptr;
```

Comment:
just notice here but I guess it's relevant to all other commits - you can drop the `struct` keyword

Reply:
Removed the unneeded `struct` keyword